tests/data/Ref_nexus_mpes.log

```diff
@@ -454,22 +447,17 @@ DEBUG - classes:
 NXmpes.nxdl.xml:/ENTRY/data/energy
 NXdata.nxdl.xml:/AXISNAME
 DEBUG - NXmpes.nxdl.xml:/ENTRY/data/energy@type - [NX_CHAR]
-DEBUG - <<REQUIRED>>
-DEBUG - enumeration (NXmpes.nxdl.xml:/ENTRY/data/energy/type):
-DEBUG - -> kinetic
-DEBUG - -> binding
+DEBUG - <<RECOMMENDED>>
 DEBUG - Dataset referenced as NXdata AXIS #2
 DEBUG - documentation (NXmpes.nxdl.xml:/ENTRY/data/energy/type):
 DEBUG - 
-                         The energy can be either stored as kinetic or as binding energy.
-                    
 DEBUG - ===== ATTRS (//entry/data/energy@units)
 DEBUG - value: eV 
 DEBUG - classpath: ['NXentry', 'NXdata', 'NX_NUMBER']
 DEBUG - classes:
 NXmpes.nxdl.xml:/ENTRY/data/energy
 NXdata.nxdl.xml:/AXISNAME
-DEBUG - NXmpes.nxdl.xml:/ENTRY/data/energy@units [NX_ENERGY]
+DEBUG - NXmpes.nxdl.xml:/ENTRY/data/energy@units - REQUIRED, but undefined unit category
```

Comment:
@rettigl & @lukaspie We also forgot to re-add the `@units` on energy after removing `NXdata_mpes` from the appdef.